submission-folder/kwangyang.ts

```diff
@@ -7,46 +7,49 @@
 
 // replace all `var` declarations appropriatly
 
-var NINE_THOUSAND = 9000
-var number = 6
+const NINE_THOUSAND = 9000
+let number = 6
 number = 7
 
 // create a type alias (rmb type alias names use PascalCase)
-var bob
-var steve
-var mary
+type Student = {
+  name: string
+  year: number
+  friends: string[]
+  favouriteVariable: string | number
+}
 
-bob = {
+const bob = {
   name: 'Bob',
   year: 2,
   friends: ['Steve'],
   favouriteVariable: 23,
 }
 
-steve = {
+const steve = {
   name: 'Steve',
   year: 1,
   friends: ['Bob', 'Mary'],
   favouriteVariable: 'hello', // note that this shall ONLY be string or number
 }
 
-mary = {
+const mary = {
   name: 'Mary',
   year: 1,
   friends: ['Steve'],
   favouriteVariable: 100,
 }
 
-var students = [bob, steve, mary] // assign an appropriate type to `students`
+let students: Student[] = [bob, steve, mary] // assign an appropriate type to `students`
 
 // convert to anonymous function
 // use string literals
 // add appropriate type declarations to function args
-function printNameAndYear(student) {
+const printNameAndYear = (student: Student) => {
   var text = student.name + ' is year ' + student.year
   console.log(text)
 }
-function introduceSelf(student) {
+const introduceSelf = (student: Student) => {
   var text =
     'Hello! I am ' + student.name + ' and my favourite variable is ' + student.favouriteVariable
```

Comment:
you are supposed to change these string concatenation syntax to string literal syntax!